zeppelin-zengine/pom.xml

```diff
@@ -257,6 +257,13 @@
       <scope>test</scope>
     </dependency>
 
+    <dependency>
+      <groupId>org.mongodb</groupId>
+      <artifactId>mongo-java-driver</artifactId>
+      <version>3.4.1</version>
```

Comment:
No worries! 

License information looks good.

Recently CI was very unstable but now it has been fixed.
Could you try rebase and see if CI becomes green?